cms/envs/aws_migrate.py

```diff
@@ -1,30 +0,0 @@
-"""
```

Comment:
@jibsheet I believe this is already done: https://github.com/edx/edx-platform/pull/11286/files#diff-8a23909bc8d201fba8525b1dd6396247R482

Did I miss something?